code/game/objects/items/storage/boxes.dm

```diff
@@ -1296,13 +1296,32 @@
 		/obj/item/stack/sheet/mineral/wood/fifty = 1,
 		/obj/item/stack/sheet/mineral/titanium/fifty = 1,
 		/obj/item/stack/sheet/mineral/uranium/fifty = 1,
-		/obj/item/stack/sheet/mineral/diamond = 1,
+		/obj/item/stack/sheet/mineral/diamond/fifty = 1,
 		/obj/item/stack/sheet/mineral/plasma/fifty = 1,
 		/obj/item/stack/sheet/mineral/gold/fifty = 1,
 		/obj/item/stack/sheet/mineral/silver/fifty = 1,
 		/obj/item/stack/sheet/mineral/bananium/fifty = 1)
 	generate_items_inside(items_inside,src)
 
+/obj/item/storage/box/materials/reduced
+	name = "Materials Box"
+	desc = "Contains most of what you would need"
+
+/obj/item/storage/box/materials/reduced/PopulateContents()
+	var/static/items_inside = list(
+		/obj/item/stack/sheet/metal = 10,
+		/obj/item/stack/sheet/glass = 10,
+		/obj/item/stack/sheet/plastic = 10,
+		/obj/item/stack/sheet/mineral/plastitanium = 10,
+		/obj/item/stack/sheet/mineral/titanium = 10,
+		/obj/item/stack/sheet/mineral/uranium = 10,
+		/obj/item/stack/sheet/mineral/diamond = 10,
+		/obj/item/stack/sheet/mineral/plasma = 10,
+		/obj/item/stack/sheet/mineral/gold = 10,
+		/obj/item/stack/sheet/mineral/silver = 10,
+		/obj/item/stack/sheet/mineral/wood = 10)
+	generate_items_inside(items_inside,src)
```

Comment:
also no wood in the ORM

Reply:
its not in orm..?

Reply:
it is outputted from the bs harvester

Reply:
when you go to a protolaithe, and try to pull resources, try to find "wood"

Reply:
wood can be used for other things, and not supposed to be put inside the ORM

Reply:
but if you insist on removal of wood then sure